doc/cephadm/host-management.rst

```diff
@@ -88,23 +88,26 @@ To drain all daemons from a host, run a command of the following form:
 
    ceph orch host drain *<host>*
 
-The '_no_schedule' label will be applied to the host. See :ref:`cephadm-special-host-labels`
+The ``_no_schedule`` label will be applied to the host. See
+:ref:`cephadm-special-host-labels`.
 
-All osds on the host will be scheduled to be removed. You can check osd removal progress with the following:
+All osds on the host will be scheduled to be removed. You can check the progress of the osd removal operation with the following command:
```

Comment:
OSDs, "progress of OSD removal"

Reply:
@zdover23 I'm merging this now, but I recommend making _osd_ uppercase in a future PR.

Reply:
"osd" is now "OSD" as of 15 Apr 2023.

Reply:
https://github.com/ceph/ceph/pull/49715 - doc/cephadm: s/osd/OSD/ where appropriate